internal/elasticsearch/ingest/pipeline.go

```diff
@@ -74,6 +75,7 @@ func SimulatePipeline(api *elasticsearch.API, pipelineName string, events []json
 	var request simulatePipelineRequest
 	for _, event := range events {
 		request.Docs = append(request.Docs, pipelineDocument{
+			Index:  "logs-generic-default",
```

Comment:
If the `event` contains `data_stream.*` values, it would be ideal to use that as the index name as that would simulate the reality more closely.